Source/options.cpp

```diff
@@ -134,34 +146,59 @@ float GetIniFloat(const char *sectionName, const char *keyName, float defaultVal
 	return (float)GetIni().GetDoubleValue(sectionName, keyName, defaultValue);
 }
 
+bool GetIniStringVector(const char *sectionName, const char *keyName, std::vector<std::string> &stringValues, const char *defaultString)
+{
+	std::list<CSimpleIni::Entry> values;
+	if (!GetIni().GetAllValues(sectionName, keyName, values)) {
+		stringValues.emplace_back(defaultString);

```

Comment:
While this matches the convention of other single-value getters this doesn't quite match the behaviour of SimpleIni for a multi-value key. GetAllValues returns with `values` cleared, we could reasonably expect `stringValues` to also be cleared and empty in that case. This allows simplifying the check to see if a default quick messages is needed.

If you want to keep the default argument maybe it'd be useful to allow passing in `nullptr` and only populate stringValues here if defaultString is non-null?

Otherwise defaultString could be a string_view.

Reply:
For now I removed the `defaultString` and didn't populate the vector.
If we later need a default value, we can add it back with a nullptr check as you suggested.